packages/ui/src/ui/pages/query-tracker/Plan/utils.ts

```diff
@@ -596,9 +596,10 @@ export function usePrepareNode(operationIdToCluster: Map<string, string>) {
         } else if (node.progress?.remoteId) {
             const id = node.progress?.remoteId.split('/').pop();
 
-            if (id && operationIdToCluster.has(id)) {
-                const cluster = operationIdToCluster.get(id)!;
-
+            const cluster = operationIdToCluster.has(id!)
```

Comment:
The id can be undefined. Now the check is disabled via !
I think it's better to add a check `if(!id) {node.url = getOperationUrl(node); return node;}`